files/logstash-configs/22-redmine.conf

```diff
@@ -23,6 +23,9 @@ filter {
         match => {"message" => "%{REDMINE_SUCCESSFUL_LOGIN}"}
         add_tag => "successful_login"
       }
+      date {
+        match => ["redmine_auth_timestamp", "yyyy-MM-dd HH:mm:ss"]
```

Comment:
@conorsch not sure... the auth lines are one-offs, while redmine_timestamp is typically associated with a multi-line. It shouldn't cause any problems to use the same field name.